packages/hash/frontend/src/components/layout/PageHeader/SearchBar/index.tsx

```diff
@@ -156,6 +210,6 @@ const SearchBarWhenSearchIsDisabled: VoidFunctionComponent = () => {
 
 // Note: This component becomes empty is opensearch is disabled
 export const SearchBar =
-  HASH_OPENSEARCH_ENABLED === "true"
+  HASH_OPENSEARCH_ENABLED !== "true"
     ? SearchBarWhenSearchIsEnabled
     : SearchBarWhenSearchIsDisabled;
```

Comment:
Why has the logic for this been changed? The FE would now do the opposite of what the environment variables tell it to do

Reply:
Understood, you can always directly toggle the value of `HASH_OPENSEARCH_ENABLED` in your local `.env`. That'll ensure things like this don't accidentally slip into our code.